test/tests/userobjects/internal_side_user_object/internal_side_user_object_two_materials.i

```diff
@@ -0,0 +1,82 @@
+[Mesh]
+  type = FileMesh
+  file = TwoBlockMesh.e
```

Comment:
Will change. BTW, if my other PR is accepted, we do not need this mesh, it is basically

```
nx=2
ny=2
subdomain_ids = '1 0
                             1 1'
```